admin_manual/configuration_server/index.rst

```diff
@@ -22,5 +22,6 @@ ownCloud Server Configuration
    search_configuration   
    thirdparty_php_configuration
    performance_tuning
+   performance_tuning/index
```

Comment:
Don't the two new docs replace performance_tuning.rst? I was planning to remove performance_tuning.rst from the server.